specification/maps/data-plane/Creator/preview/2022-09-01-preview/wayfind.json

```diff
@@ -222,22 +230,25 @@
     },
     "AvoidFeatures": {
       "name": "avoidFeatures",
-      "description": "To avoid certain types of connectors in the resulting path. Multiple values can be specified.",
-      "type": "array",
-      "items": {
-        "type": "string",
-        "enum": [
-          "stairs",
-          "elevators"
-        ],
-        "x-ms-enum": {
-          "name": "avoidFeatures",
-          "values": [
-            "stairs",
-            "elevators"
-          ],
-          "modelAsString": true
-        }
+      "description": "To avoid certain types of connectors in the resulting path. Can be specified multiple times in one request, for example, '&avoidFeatures=elevators&avoidFeatures=escalators'.",
+      "type": "string",
```

Comment:
Don't change this to a string. If the user can specify it multiple times, it should be an array as it was originally. The part you are missing is 'collectionFormat: multi', which specifies that multiple values repeat the parameter name.  See

https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#parameter-object

specifically

<img width="877" alt="image" src="https://user-images.githubusercontent.com/85643503/233632832-8c519bda-4fdb-4a48-9451-a7ba8b11926f.png">

Reply:
thank you, Mike!

Reply:
The error I receive pertains to a missing schema element
ERROR: Schema violation: Data does not match any schemas from 'oneOf'\n - 
file:///mnt/vss/_work/1/same-version-c93b354fd9c14905bb574a8834c4d69b/specification/maps/data-plane/Creator/preview/2022-09-01-preview/wayfind.json:223:4 
($.parameters.AvoidFeatures)\nFATAL: swagger-document/individual/schema-validator - 

a) The error references line 223, at which AvoidFeatures is declared in the original file. If the error is caused by the original file, there is no way for me to bypass it.

b) Here is a condensed side by side of the 3 param declarations. I don't see a difference in the way it is modeled from v1.0, which, being in main, I assume passed the verification. It also failed when I modeled it after v2023-08-01 (string instead of array).

any chance this error is a false positive?
 
![image](https://user-images.githubusercontent.com/88048110/233735790-b13a9ad6-8507-4ce1-a276-611bb638a717.png)

Reply:
more details on the error
![image](https://user-images.githubusercontent.com/88048110/233738732-958c34e8-e7df-47f6-aa94-09d2d6d02a45.png)